### DISCUSSION TITLE
Unable to verify the first certificate with n8n

### DISCUSSION THREAD

**Discussion by charlesdsmith** [Budibase/budibase#12359] [Help]
I am trying to send a GET request to our n8n webhook address, and we get the following error:
![image](https://github.com/Budibase/budibase/assets/38652599/8a92c42b-92f3-46ea-94e6-98f3d6d286f7)

I have no issue sending a GET request to the address via Chrome. Therefore I don't believe there's an issue with my certs, any ideas?

**Comment by sdsys-ch** [comment]
Hi @charlesdsmith , node doesn't like certificates that don't include the entire certificate chain while browsers have a tendency to complete them on their own. Debug yours with e.g. 

`openssl s_client -showcerts -connect n8n.en-pro.com:443 -servername n8n.en-pro.com`

or any online tool to check ssl certs.

Yours is apparently issued by godaddy but doesn't include chain certs 1 and 2.

Best fix: have the server serve a pem certificate that includes the entire chain. Not sure how to do that with godaddy issued certs.

Alternative if you control the node env: fetch all chain certs and add them to yours in one cert (full chain.pem) and then look at setting `NODE_EXTRA_CA_CERTS` environment variable for a fix.

Hope this helps.
